src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
Actual TFM (`netcoreapp2.0` at the time) was previously removed because @maxkatz6 complained about some build problems. Not sure if those are still there, but it's safer to leave it as is.

Reply:
Removed, this part is quite easy to add manually.